### PR TITLE
Teet-981 changes

### PR DESCRIPTION
**Pull request by solita-erkkiv**
The new styles for meetings and links are missing but they can be created separately if needed.

### CODE REVIEW

File: app/frontend/src/cljs/teet/file/file_view.cljs [hunk 1]
```diff
@@ -332,8 +343,7 @@
            [buttons/delete-button-with-confirm
             {:action #(delete-action file)
              :trashcan? true}])]
-        (when (and comments-link? (:comment/counts file))
-          [file-comments-link file])])]))
+        ])]))
```
Comment (jkorpelin) [inline app/frontend/src/cljs/teet/file/file_view.cljs:346]: This shouldn't be it's own row

File: app/frontend/src/cljs/teet/file/file_view.cljs [hunk 2]
```diff
@@ -304,8 +312,11 @@
                  {:date (format/date-time (:meta/created-at file))
                   :author (user-model/user-name
                             (:meta/creator file))}))]
-        (when status
-          [typography/SmallBoldText (tr-enum status)])]]]
+        [:div {:class [(<class common-styles/flex-align-center)]}
```
Comment (jkorpelin) [inline app/frontend/src/cljs/teet/file/file_view.cljs:315]: does this work? I thought that you had to have :classes if you wanted to provide a list of classes to an element?

File: app/frontend/src/cljs/teet/link/link_view.cljs [hunk 3]
```diff
@@ -113,15 +126,19 @@
                      link-entity-opts
                      {id :db/id
                       :link/keys [to type] :as link}]
-  [:div {:class [(<class common-styles/flex-row-space-between)]}
-   [:div {:style {:flex :auto
-                  :min-width 0}}
+  [:div {:class [(<class common-styles/flex-row-space-between)]
+         :style {:border-top (str "solid " theme-colors/gray-lighter " 2px")}}
+   [:div {:style (merge {:flex :auto
```
Comment (jkorpelin) [inline app/frontend/src/cljs/teet/link/link_view.cljs:131]: this could be indented better, or better yet made into it's own class.

**Review by jkorpelin — COMMENTED** [review]
image

should the add decision look like this?

File: app/frontend/src/cljs/teet/meeting/meeting_view.cljs [hunk 4]
```diff
@@ -150,6 +153,62 @@
...
+                (when (or children after-children-component)
+                  [Collapse {:in @open?
+                             :mount-on-enter true}
+                   [:div {:class (<class common/hierarchical-child-container)
+                          :style {:border-top (str "1px solid " theme-colors/gray-lighter)}}
+                    (doall
+                      (for [child children]
+                        (with-meta
+                          (if (vector? child)                         ;;Check if it's component and render that instaed
+                            child
```
Comment (jkorpelin) [inline app/frontend/src/cljs/teet/meeting/meeting_view.cljs:207]: Is all of this stuff necessary? At a quick glance I can't find a place where this component gets a component as a child. So some of these checks could be removed.

File: app/frontend/src/cljs/teet/ui/tabs.cljs [hunk 5]
```diff
@@ -21,6 +22,32 @@
...
+   :padding "13.5px 8px"
+   :background "#005AA3"
+   :border-radius "3px 3px 0px 0px"})
+
+(defn tab-links-container-style
+  []
+  {:display :flex
+   :margin-bottom "2rem"
+   :padding-bottom "0"
+   :border-bottom "solid #005AA3 1px"})
```
Comment (jkorpelin) [inline app/frontend/src/cljs/teet/ui/tabs.cljs:49]: these colors should come from theme-colors

File: app/frontend/src/cljs/teet/meeting/meeting_view.cljs [hunk 6]
```diff
@@ -881,27 +952,60 @@
...
+                                                                                                    :user/email
+                                                                                                    :user/person-id])}
+                                  :modal-title      (tr [:meeting :new-agenda-modal-title])
+                                  :button-component [buttons/button-primary {:start-icon (r/as-element
+                                                                                           [icons/content-add])} (tr [:meeting :add-agenda-button])]}])]
      [:div {:style {:margin-bottom "1rem"}}
       (doall
        (for [{id :db/id
-              :meeting.agenda/keys [topic responsible body decisions] :as agenda-topic} agenda]
+              :meeting.agenda/keys [topic responsible body decisions file/_attached-to] :as agenda-topic} agenda]
```
Comment (tatut) [inline app/frontend/src/cljs/teet/meeting/meeting_view.cljs:972]: Cant use `file/_attached-to` in `:meet.agenda/keys`... that would make the key `:meeting.agenda/file/_attached-to` which is not correct.

This needs to be as separate destructuring in the parent map.

**Review by tatut — CHANGES_REQUESTED** [review]
See destructuring problem in comment

**Pull request merged**